Few tweaks to astro config.

nikoshell opened the pull request:
## Configuration Improvements

### Changes to `astro.config.mjs`:
- Added build-time environment variable definition
- Configured Vite alias for `$` to resolve from `./src`
- Enabled minification for production builds
- Added path resolution for `__dirname` in ES module context

### Updates to `tsconfig.json`:
- Added TypeScript path alias for `$*` to map to `src/*`

<!-- readthedocs-preview ep-website start -->
🖼️ Preview available 🖼️ : https://ep-website--1068.org.readthedocs.build/
<!-- readthedocs-preview ep-website end -->

nikoshell committed:
Few tweaks to astro config.

europython-society commented:
# Preview available
|Key|Value|
|---|---|
|url|https://ep2025-core.ep-preview.click|
|last update|2025-03-29T02:36:52.744Z|

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

egeakman commented:
Thanks for the PR! Can you add some description telling what the changes do in the original comment?

nikoshell commented:
## Configuration Improvements

### Changes to `astro.config.mjs`:
- Added build-time environment variable definition
- Configured Vite alias for `$` to resolve from `./src`
- Enabled minification for production builds
- Added path resolution for `__dirname` in ES module context

### Updates to `tsconfig.json`:
- Added TypeScript path alias for `$*` to map to `src/*`
- Maintained existing TypeScript and React configuration

nikoshell committed:
Merge branch 'ep2025' into ep2025-core

nikoshell committed:
Merge branch 'ep2025' into ep2025-core

nikoshell committed:
Add environment param SITE_URL.

nikoshell committed:
Merge branch 'ep2025' into ep2025-core

nikoshell committed:
Add paths to tsconfig.

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

egeakman committed:
Merge branch 'ep2025' into ep2025-core

egeakman reviewed (approved):
Thank you!
